PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -114,6 +114,9 @@ static WORD GetBinaryArchitectureType(const TCHAR *filePath)
 	return machine_type;
 }
 
+#define	LOAD_LIBRARY_SEARCH_DLL_LOAD_DIR	0x00000100
+#define	LOAD_LIBRARY_SEARCH_DEFAULT_DIRS	0x00001000

```

Comment:
It's strange that you have to define 2 symbols yourself - is there no header files to be included ?

Reply:
nm, I read the previous comment.